cluster/gce/container-linux/configure-helper.sh

```diff
@@ -459,6 +460,8 @@ function start-kubelet {
   flags+=" --cluster-domain=${DNS_DOMAIN}"
   flags+=" --config=/etc/kubernetes/manifests"
   flags+=" --experimental-check-node-capabilities-before-mount=true"
+  flags+=" --require-kubeconfig"
```

Comment:
hold until https://github.com/kubernetes/kubernetes/pull/40050#issuecomment-274117412 is resolved